internal/pkg/manifest/env_test.go

```diff
@@ -509,6 +504,36 @@ http:
 				},
 			},
 		},
+		"unmarshal with new http fields": {
```

Comment:
Let's also add a use case that unmarshal an "invalid" manifest that has both old ingress and new ingress and see how it unmarshals.
```yaml
http:
  public:
     ingresS:
       cdn: true
     security_groups:
        ingress:
           cdn: true
```